src/commands/CountryCommand.ts

```diff
@@ -0,0 +1,391 @@
+import { Command } from "../Command";
+import { CommandInteraction, MessageEmbed } from "discord.js";
+import {
+	inlineCode,
+	SlashCommandBuilder,
+	SlashCommandStringOption,
+	SlashCommandSubcommandsOnlyBuilder,
+} from "@discordjs/builders";
+import {
+	Country,
+	countryData,
+	CountryKey,
+	getCountryByName,
+	getFilteredCountryDataBy,
+	initializeCountryData,
+	sortCountryDataBy,
+	typeOfCountryProperty,
+} from "../util/countryDataManager";
+
+export let formatNumber: (n: number) => string;
```

Comment:
```suggestion
```

Reply:
the funciton itself should/could be used in different contexts too. though should be moved to a diffent file